<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_ThorConfig</class>
 <widget class="QDialog" name="Dial_ThorConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thor Configuration</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>File Select</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_Fname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Select">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do It</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>461</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
